### ISSUE TITLE
Early epiphyseal ossification

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11364]
**Preferred term label:**

**Synonyms**
Premature epiphyseal closure
Premature epiphyseal ossification
Premature fusion of the epiphyseal growth plate

**Definition (free text, please give PubMed ID)**
PMID: 22024975 The effect of aromatase inhibitors on growth appears to be mediated primarily via reduced estrogen production within the epiphyseal chondrocytes, but a decrease of circulating estrogens also affects growth hormone (GH) and insulin-like growth factor 1 (IGF‑1) secretion. The current view is that, in men as well as women, estrogen does not participate in the regulation of linear growth before puberty but plays a major part in inducing the pubertal growth spurt (at low levels) and in **epiphyseal maturation and closure** (at high levels).
(...) Although prolonged intervals of quiescence occur between vaginal bleeding episodes, some girls with McCune–Albright syndrome experience 
progression of precocious puberty with frequent menstrual bleeding, associated development of other secondary sexual characteristics, acceleration of linear growth and advancement of skeletal maturation leading to **early closure of the epiphyseal growth plates** and compromised adult height.

 PMID: 34538723 Estrogen excess during the prepubertal period initially accelerates bone growth and bone age, and subsequently, induces **premature epiphyseal closure**

PMID: 33513243 Boys with AEXS present with prepubertal gynecomastia at the age of adrenarche when serum dehydroepiandrosterone (DHEA) reaches a concentration sufficiently high for the production of estrone and  estradiol amounts capable to induce growth of the mammary glands. Later, during puberty, the excess of estrogens causes inhibition of normal testicular growth and virilization in boys by suppression of gonadotropins. 
Concomitantly, acceleration of height velocity and bone maturation is observed, which can compromise adult height by **early epiphyseal closure** and premature growth arrest.
(...) AEXS can cause adult short stature by premature fusion of the epiphyseal growth plate due to excessive production of estrogens. 

**Parent term (use hpo.jax.org/app)**
Just as there is a group called Delayed epiphyseal ossification HP:0002663, I suggest creating this group into **Abnormal epiphyseal ossification HP:0010656**, which includes, among others, the terms: Premature fusion of phalangeal epiphyses HP:0006140, Early ossification of capital femoral epiphyses HP:0008797, etc.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Aromatase excess syndrome (AEXS) ORPHA:178345

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@azankl
I am wondering that since there is no specific epiphysis mentioned here if this term is essentially the same as
[Accelerated skeletal maturation HP:0005616](https://hpo.jax.org/browse/term/HP:0005616), because this is measure with a hand X ray usually. Is there a utility in having both terms?

**Comment by azankl** [comment]
@pnrobinson agree, the way it is used in the examples above, it is identical to [Accelerated skeletal maturation HP:0005616](https://hpo.jax.org/browse/term/HP:0005616). Maybe we could add `advanced epiphyseal ossification` and `premature fusion of growth plates` as synonyms to [Accelerated skeletal maturation HP:0005616](https://hpo.jax.org/browse/term/HP:0005616)?

**Comment by pnrobinson** [comment]
@azankl @carolina-f -- I have added the above as synonyms, thanks Andreas for the clarification!
